src/components/PieChartInfographic.jsx

```diff
@@ -0,0 +1,566 @@
+          })
+
+          var final = [];
+
+
+          // transpose..... list of points in a series to a list of points for a timestamp
+          var findMax = []
+          var rows = []
+          for (var i = 0, len_i = items[0].length; i < len_i; i++) {
+            var row = []
+            for (var j = 0, len_j = items.length; j < len_j; j++) {
+              var val = items[j][i] || 0
+              row.push(val)
+              findMax.push(val)
+            }
+            rows.push(row)
+          }
+          dataKeys.forEach( (key, i) => {
+            var int = parseInt(key)
+            if(int > 0){
+              final.push([int].concat(rows[i])) 
+            }
+          });
+
+
+          var series = new TimeSeries({
+            name: "timeseries",
+            columns: ["time"].concat(columns),
+            points: final.sort( (a,b) => a[0] - b[0])
+          })
+
+          // set categories
+          var categories = columns.map(column => {
+            return {
+              key: column,
+              label: column[0].toUpperCase() + column.slice(1,column.length),
+              value: null
+            }
+          })
+          var vals = columns.map((column,idx)=> {
+            return {
+              key: column,
+              color: that.props.infographicDefinitions.lineChartConfig.customColorsList[idx],
+              width: 2
+            }
+          })
+          // set style according to categories
+          var legendStyle = styler(vals)
+
+
+          that.setState({
+            legendStyle,
+            categories,
+            timerange: new TimeRange(new Date(series.toJSON().points[0][0]), that.state.maxTime),
+            selectedClassName: that.props.infographicDefinitions.defs[data.id],
+            sparklineData: series,
+            sparklineDataMax: Math.max(...findMax),
+            lineChartColumns: ["time"].concat(columns),
+            classSelection: data.id
+          })
+
+          this.onSelectionChange(categories[0].key)
+          
+          var vals = $("text").filter(function () {
+              return $(this).attr("transform") == "rotate(-90)"
+          })
+          if(vals.length){
+            $(vals[0]).attr("x",that.props.infographicDefinitions.xLengendCoordinate)
+          }
+        })
+      })
+  }
+
+  onHighlightChange (highlight){
+    // console.log(highlight)
+    // this.setState({ highlight })
+  }
+
+  onSelectionChange(selection) {
+    var selectionName = this.props.infographicDefinitions.selectionPostFix !== undefined ? 
+                        selection +  this.props.infographicDefinitions.selectionPostFix : 
+                        selection;
+
+    this.setState({ 
+      selection,
+      selectionName: selectionName
+    })
+
+    // calculate size of div for tooltip, add 35 for numbers
+    // const size = calculateSize(selectionName, this.props.infographicDefinitions.tooltip)
+    // $($('rect')[2]).css("width",size.width+35)
```

Comment:
There are a lot of commented out lines of code in this file. If we don't need this code anymore, we can remove these. If it might be useful in the future we can certainly keep it though.